packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -49,8 +49,8 @@ If you already have ngrok installed, move on to [Run ngrok](#run-ngrok). If not,
 
 Some installation notes:
 
-* You don't need to have an account to install and run ngrok, but creating a free account provides more features as well as basic authentication.
-* You can install ngrok directly in your project folder, sample-app, as documented from the ngrok download page. Or you can install on your system's path directory to be able to run ngrok from any folder. Alternatively, you can install the executable in your favorite local folder, but you'll need the folder path when referencing the tool.
+* You don't need to have an account to install and run ngrok, but creating a free account provides more features including basic authentication.
+* You can install ngrok directly in your project folder, sample-app, as documented from the ngrok download page. Or you can install on your system's path directory to be able to run ngrok from any folder. Alternatively, you can install the executable in your favorite local folder, but you need the folder path when referencing the tool.
```

Comment:
This seems a little clunky.

What about something like this:
`You can install ngrok in the system path directory or the project directory. Alternatively, you can install the executable in another local directory, as long as you reference the direcotyr path when using the tool.`